qcore/config.py

```diff
@@ -1,22 +1,34 @@
-
 import json
 import os
 import platform
 
-hostname = platform.node()
-if hostname.startswith("ni") and len(hostname) == 8:
-    host = "maui"
-    basename = os.path.join('machine_config', 'config_maui.json')
-elif hostname.startswith("wb") and len(hostname) == 6:
-    host = "mahuika"
-    basename = os.path.join('machine_config', 'config_mahuika.json')
-else:
-    # default
-    host = "unknown"
-    basename = 'config.json'
 
+def get_host_config():
+    """
+    Determine the actual hostname eg.ni0002--> maui; maui01-->maui
```

Comment:
the actual hostname is the result of platform.node could say machine name or something